pyproject.toml

```
@@ -0,0 +1,7 @@
+[build-system]
+requires = ["setuptools>=44", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

```suggestion
requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

You only need 42 or better, and 44 kills Python 3.4. Not that I'm worried about 3.4, but if it's easy to fix here, we might as well.

---

You should add `pathlib;python_version<"3"` here too, since it's needed for setup.py.  This won't fix Python 2.6, since 2.6 does not have a new enough pip installed to respect this file. But that's okay, we can install pathlib manually in CI.

None of this affects the wheel, which is what 100% of users install.

---

It turned out that we don't need pathlib anymore since we have moved the stuff about `setuptools_scm` to `pyproject.toml`.